sdk/blockchain-client-provider/src/interfaces/index.ts

```diff
@@ -0,0 +1 @@
+export { type IBlockchainClientProvider } from './IBlockchainClientProvider'
```

Comment:
Suggestion: I found that it works better to do:
```
export * from './IBlockchainClientProvider'
```

As there are less updates to be done when something is added to IBlockchainClientProvider. It is used all across the SDK, but let me know if you disagree

Reply:
I think that is nice solution, I don't see any issues with that :)